api/src/main/java/org/openmrs/Obs.java

```diff
@@ -153,6 +156,9 @@ public enum Status {
 	private Interpretation interpretation;
 	
 	private Status status = Status.FINAL;
+
+	@OneToOne(mappedBy = "obs", cascade = CascadeType.ALL)
```

Comment:
I do not recommend touching the Obs class for something that affects only numeric obs.